layers/extensions/dual_wiper_systems.vspec

```diff
@@ -0,0 +1,32 @@
+#
+# (C) 2022 Robert Bosch GmbH
+#
+# Overlay to extend wiper system to support multiple instances
+# Dependencies to other overlays: None
```

Comment:
Two general things I think we should discuss:

- As of today we sometimes use the term "overlay", sometimes "layer". Shall we use both, and if so when shall which term be used?
- Shall we have some general comment syntax like here to specify known dependencies and/or conflicts. We may e.g. have a layer that only can be applied on top of the motorbike profile/layer.